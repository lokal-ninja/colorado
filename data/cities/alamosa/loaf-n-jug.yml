id: 4017450845
type: node
latitude: 37.4736
longitude: -105.85674
postcode: "81101"
city: Alamosa
street: US-160
housenumber: "102"
phone: ""
opening_hours: ""
website: ""
