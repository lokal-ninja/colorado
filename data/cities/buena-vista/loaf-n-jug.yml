id: 130574087
type: way
latitude: 38.83541
longitude: -106.12933
postcode: "81211"
city: Buena Vista
street: "US Highway 24"
housenumber: 610
phone: "+1-719-395-2121"
opening_hours: "24/7"
website: ""