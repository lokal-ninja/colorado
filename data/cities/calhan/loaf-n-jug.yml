id: 800115114
type: way
shop: convenience
latitude: 39.0363212
longitude: -104.3044789
postcode: "80808"
city: Calhan
street: "5th Street"
housenumber: 1025
phone: ""
opening_hours: ""
website: ""
