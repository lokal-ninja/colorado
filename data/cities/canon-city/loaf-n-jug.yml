id: 387732276
type: way
shop: convenience
latitude: 38.442393200000005
longitude: -105.2341957
postcode: "81212"
city: Ca√±on City
street: "Royal Gorge Boulevard"
housenumber: 912
phone: ""
opening_hours: "24/7"
website: ""
