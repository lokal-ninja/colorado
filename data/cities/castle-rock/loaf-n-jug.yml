id: 222904661
type: node
latitude: 39.40986
longitude: -104.87242
postcode: "80109"
city: Castle Rock
street: North Castleton Drive
housenumber: "4901"
phone: +1-303-663-9020
opening_hours: 24/7
website: https://www.loafnjug.com/
