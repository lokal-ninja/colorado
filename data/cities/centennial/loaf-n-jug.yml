id: 352693599
type: node
latitude: 39.61339
longitude: -104.75663
postcode: "80015"
city: Centennial
street: East Smoky Hill Road
housenumber: "19853"
phone: ""
opening_hours: ""
website: ""
