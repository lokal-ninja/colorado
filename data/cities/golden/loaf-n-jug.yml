id: 946482125
type: node
latitude: 39.7645488
longitude: -105.2329983
postcode: "80403"
city: Golden
street: "North Rubey Drive"
housenumber: 102
phone: "+1-303-215-0883"
opening_hours: "24/7"
website: "https://www.loafnjug.com/"
